sensor/common/image/service_impl.go

```diff
@@ -58,14 +64,16 @@ func (s *serviceImpl) GetImage(ctx context.Context, req *sensor.GetImageRequest)
 		}
 	}
 
+	log.Debugf("scan request from admission control: %+v", req)
+
 	// Note: The Admission Controller does NOT know if the image is cluster-local,
 	// so we determine it here.
 	// If Sensor's registry store has an entry for the given image's registry,
 	// it is considered cluster-local.
 	req.Image.IsClusterLocal = s.registryStore.HasRegistryForImage(req.GetImage().GetName())
```

Comment:
Can you comment here that this "clsuter-local" stuff is really just about OpenShift Internal registry at this time

Reply:
will do